src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -206,6 +207,54 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     fromAddress: fromAddress ?? null,
   })
 
+  const { amountCryptoBaseUnitMinusFees, amountCryptoHumanMinusFees } = useMemo(() => {
+    if (!estimatedFeesData)
+      return {
+        amountCryptoBaseUnitMinusFees: undefined,
+        amountCryptoHumanMinusFees: undefined,
+      }
+
+    if (
+      state?.deposit.sendMax &&
+      estimatedFeesData &&
+      !isUtxoChainId(feeAsset.chainId) &&
+      feeAsset.assetId === asset.assetId
+    ) {
+      return {
+        amountCryptoBaseUnitMinusFees: bn(
+          toBaseUnit(state?.deposit.cryptoAmount, asset.precision),
+        ).minus(estimatedFeesData.txFeeCryptoBaseUnit),
+        amountCryptoHumanMinusFees: fromBaseUnit(
```

Comment:
This is a precision amount i.e full precision
```suggestion
        amountMinusFeesCryptoPrecision: fromBaseUnit(
```

Reply:
Fixed in [785cec8](https://github.com/shapeshift/web/pull/7606/commits/785cec859251e10c00102f88be7c1bab7d28963f)